workflow-templates/check-shell-task.yml

```diff
@@ -144,7 +144,7 @@ jobs:
 
       - name: Download shfmt
         id: download
-        uses: MrOctopus/download-asset-action@1.0
+        uses: MrOctopus/download-asset-action@1.1
```

Comment:
An alternative approach to installation would be using `go install` (which was not available for use at the time the template was written). This is an officially supported installation method for the tool:

https://github.com/mvdan/sh#shfmt

My personal preference would be to use that installation approach in combination with [a dummy "tools" Go package](https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module), which would allow controlled bumps of the tool dependency via automated Dependabot PRs. However, we could also just use `go install mvdan.cc/sh/v3/cmd/shfmt@latest` to get simple uncontrolled installation as is done with the current approach.